app/templates/index.html

```diff
@@ -18,7 +22,6 @@
     >
```

Comment:
Tiny thing and not related to this PR, but I’m getting a warning from my [Tailwind editor extension](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) about the CSS classes here:

> 'w-full' applies the same CSS properties as 'w-screen'.

So, one of the classes can probably leave this code base! 🫡